modules/custom/az_news/az_news_feeds/src/EventSubscriber/AzNewsFeedsMigrateSubscriber.php

```diff
@@ -32,13 +32,14 @@ public function __construct(
   }
 
   /**
-   * Helper method to check if we are migrating UArizona News stories.
+   * Helper method to check if we are migrating University News stories.
```

Comment:
Doesn't really matter since it's just a code comment, but "University News" is used here while "University of Arizona News" is used right below. It should either be lowercase u or changed to "University of Arizona" like other references.

Reply:
Good catch! I didn't use the full "University of Arizona" here since it would make the line too long and PHPCS would complain, but I like the idea of changing it to lowercase.

Reply:
```suggestion
   * Helper method to check if we are migrating university news stories.
```